### ISSUE TITLE
Automated emails for reviewers at the end of a successful review

### ISSUE THREAD

**Issue by arfon** [openjournals/joss#1386]
When a review completes, we could send out an automated email to reviewers thanking them for their review. This would presumably(?) satisfy the need that some reviewers have for a 'reviewer certificate' for completed JOSS reviews.

How this could work:

When the paper is moved to `accept`...
- For each reviewer (identified by their GitHub handle)
- Look up the reviewer on the JOSS reviewers application
- If they have an account their, retrieve the email address.
- Send an email.

**Comment by sneakers-the-rat** [comment]
i can help with dis. did you have a dev plan for this already or do ya wanna work on this together?

**Comment by danielskatz** [comment]
While I don't really want to expand the checklist, I wonder if we could add an item for the reviewer to opt-in to this, and use it to decide whether to send out an email or not. I don't really want to send more emails to JOSS reviewers who don't want them...

**Comment by sneakers-the-rat** [comment]
i was thinking the same thing with : https://github.com/openjournals/joss/issues/1387

i always forget about the reviewer profile, so agreed that adding some checkbox to the reviewer options to enable/disable emails would be great :)

**Comment by kthyng** [comment]
I agree that the reviewer profile is a good place for this if we require opt-in. But is one email for a review so bad @danielskatz?
